docs/csharp/language-reference/operators/arithmetic-operators.md

```diff
@@ -30,12 +30,12 @@ helpviewer_keywords:
 ---
 # Arithmetic operators (C# reference)
 
-The following operators perform arithmetic operations with numeric types:
+The following operators perform arithmetic operations with the operands of numeric types:
```

Comment:
To be more accurate for unary operators.

```suggestion
The following operators perform arithmetic operations with the operand(s) of numeric types:
```

Reply:
I think current wording is fine, though not perfectly accurate. To me, those parentheses look rather distracting than helping.